### PR TITLE
Cluster responsive

### PR DESCRIPTION
**Pull request by jyash97**
- [x] Requires Approval of #83 to make Cluster Info Page Responsive.

### CODE REVIEW

**Comment by jyash97** [comment]
Oh okay, yeah I have mentioned this in comment, needed approval of #83

**Comment by siddharthlatest** [comment]
#83 is already merged. You can update the PR to include those changes as well. (I noted the above issue after including those changes)

**Comment by jyash97** [comment]
Fixed

**Comment by siddharthlatest** [comment]
@jyash97 can you align the bottom area as well?

**Comment by jyash97** [comment]
@siddharthlatest  You can check now if everything looks good

**Comment by metagrover** [comment]
@jyash97 

- We should show the title and content in separate rows to utilise the spacing better.

image

- We should make the credentials UI responsive.

image

**Comment by jyash97** [comment]
Fixed the spacing between title and Radio buttons 
screen shot 2018-10-10 at 6 31 34 pm

Not getting how to make the credentials UI responsive, though we can scroll in x direction to view full credential

File: src/pages/ClusterPage/components/PricingSlider/index.js [hunk 1]
```diff
@@ -72,15 +72,15 @@ export default class PricingSlider extends Component {
 
 		return (
 			<Fragment>
-				<div className="col grow" style={{ padding: '55px 70px 0 55px' }}>
+				<div className="col grow pricing">
```
[the remark this answers is not in the document]
Comment (jyash97) [reply src/pages/ClusterPage/components/PricingSlider/index.js]: Agreed. Will keep it in mind 👍

**Comment by metagrover** [comment]
Merged!

**Pull request closed without merging**